mutiny-wasm/src/lib.rs

```diff
@@ -1273,7 +1293,8 @@ impl MutinyWallet {
     #[wasm_bindgen]
     pub fn get_nwc_uri(&self, index: u32) -> Result<String, MutinyJsError> {
         match self.inner.nostr.get_nwc_uri(index) {
-            Ok(uri) => Ok(uri.to_string()),
+            Ok(Some(uri)) => Ok(uri.to_string()),
+            Ok(None) => Ok("NWA".to_string()),
```

Comment:
We're going to need some changes on the frontend side for this, right? Wouldn't it be better to just leave string as empty?